src/main/java/com/epam/reportportal/junit/ReportPortalListener.java

```diff
@@ -72,15 +72,17 @@
  * launch should be unique. (User shouldn't run the same classes twice/or more
  * times in the one launch)
  */
-public class ReportPortalListener implements ShutdownListener, RunnerWatcher, RunWatcher<FrameworkMethod>, MethodWatcher<FrameworkMethod> {
```

Comment:
The type parameter was removed from **AtomicTest** and **RunWatcher**, because I determined these classes didn't warrant being generic.